config/zones/TR.yaml

```diff
@@ -120,6 +124,12 @@ emissionFactors:
       datetime: '2021-01-01'
       source: INCER ACV
       value: 26.15
+    geothermal:
+      datetime: '2020-01-01'
```

Comment:
The datetime listed in the paper for the value you added is August 2019, not January 2020.